.github/workflows/workflow.yaml

```diff
@@ -0,0 +1,90 @@
+name: Pipeline - Continuos Integrations
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+      - test
+      - 'feature/**'
+      - 'feat/**'
+      - 'fix/**'
+      - 'bugfix/**'
+      - 'hotfix/**'
+  pull_request:
+    branches:
+      - main
+      - dev
+      - test
+      - 'feature/**'
+      - 'feat/**'
+      - 'bugfix/**'
+      - 'hotfix/**'
+
+jobs:
+
+  build:
+    name: Build Project
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [20.17.0]
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+
+    - name: Verify Node.js version
+      run: |
+        if [ "$(node -v)" != "v${{ matrix.node-version }}" ]; then
+          echo "Node.js version mismatch. Expected v${{ matrix.node-version }} but got $(node -v)"
+          exit 1
+        fi
+
+    - name: Install dependencies
+      run: npm install
+
+    - name: Build project
+      run: npm run build
+
+  lint:
+    name: Lint 
```

Comment:
colocar como nome desse job Lint and test

Reply:
Feito